components/esp_modem/test/README.md

```diff
@@ -0,0 +1,46 @@
+# ESP-Modem Testing
+
+This folder contains automated and manual tests for esp-modem component. Beside these tests, some jobs are executed in CI to exercise standard examples (please refer to the CI definition and CI related sdkconfigs in examples).
+
+List of test projects:
+
+* `host_test` -- esp_modem is build on host (linux), modem's terminal in mocked using Loobpack class which creates simple responders to AT and CMUX mode. This test is executed in CI.
+* `target`  -- test executed on target with no modem device, just a pppd running on the test runner. This test is executed in CI.
+* `target_ota` -- Manual test which perform OTA over PPP.
+* `target_iperf` -- Manual test to measure data throughput via PPP.
+
+## Manual testing
+
+Prior to every `esp_modem` release, these manual tests must be execute and pass:
```

Comment:
Is it possible to have some auto reminder on PR?

Reply:
Good idea, I'll think about it and check if we can generate some TODO list on PRs with bump commits

Reply:
I've checked that it's possible to read PR description and check for certain text. This way we could enforce a task list with marked `manual test` field for all PRs that contain a bump commit. Additionally we could provide a PR template including this task list. The drawback is that we're in mono repo and the only component that needs manual testing is esp_modem. Also we would need to document this process.
Considering the advantages and effort, I've decided to create a tracker IDF-9074 to migrate the manual tests to CI.

Please resolve if you agree.